### ISSUE TITLE
Cannot invoke 'new' on Nanobar using requireJS

### ISSUE THREAD

**Issue by westonkjones** [jacoborus/nanobar#45]
I am unable to invoke a new Nanobar object when using RequireJS. Here is an example implementation:

```
var Nanobar = require('nanobar') 
var nanobar = new Nanobar();
nanobar.go(50)
```

The result is _Uncaught TypeError: Nanobar is not a constructor_
However the following will correctly render nanobar:
```
var Nanobar = require('nanobar')
Nanobar.go(50)
```

With this implementation, however, I am unable pass options to it (need to set the target)

**Comment by jacoborus** [comment]
I'm not close to my computer right now, can you try changing line [138](https://github.com/jacoborus/nanobar/blob/93532311353a0920ec01393d39c6f2374c768f87/nanobar.js#L138) in nanobar source please?

Instead:
```js
    define([], Nanobar)
```
try:
```js
    define([], function () {return Nanobar})
```

**Comment by westonkjones** [comment]
Ah, yes. This seems to fix the issue just fine. Thanks a million @jacoborus

**Comment by jacoborus** [comment]
Fixed in v0.4.2  17b4e1c3db1e2a54010961d59ce65f78baf0d3cf

**Comment by jacoborus** [comment]
@westonkjones  you can upgrade now. Thanks for pointing the bug
